lib/reference/properties.mjs

```diff
@@ -610,6 +610,38 @@ export const longhandSubPropertiesOfShorthandProperties = new Map([
 	],
 ]);
 
+export const hasCollapsibleValues = new Set([
```

Comment:
[ask] `has***` seems to return a boolean. And, `***Properties` seems more consistent than `***Values` in this module. Do you have any other better name?

Reply:
> Do you have any other better name?

I am a hard time finding something better.
`import { hasCollapsibleValues } from '../../reference/properties.mjs';` looks OK to me.
Since it's not being used by another rule yet, should I inline the set directly in the file?
i.e. `const SUPPORTED_SHORTHANDS = new Set([…`